### PR TITLE
Feature/iamse 1531

### PR DESCRIPTION
**Pull request by scalding**
External authentication plugin

### CODE REVIEW

File: grouper-container/grouper-uiDaemon-container/pom.xml [hunk 1]
```diff
@@ -28,47 +28,52 @@
     <dependency>
```
Comment (scalding) [inline grouper-container/grouper-uiDaemon-container/pom.xml:28]: might not need this file changed

File: grouper-misc/grouper-authentication/plugin-pom.xml [hunk 2]
```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (scalding) [inline grouper-misc/grouper-authentication/plugin-pom.xml:1]: might not need this file

File: grouper-misc/grouper-authentication/src/main/java/edu/internet2/middleware/grouper/authentication/plugin/ConfigUtils.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
+package edu.internet2.middleware.grouper.authentication.plugin;
```
Comment (scalding) [inline grouper-misc/grouper-authentication/src/main/java/edu/internet2/middleware/grouper/authentication/plugin/ConfigUtils.java:1]: 2 space indentations

File: grouper/src/grouper/edu/internet2/middleware/grouper/util/GrouperUtil.java [hunk 4]
```diff
@@ -1122,7 +1122,7 @@ public static String byteCountToDisplaySize(long size) {
    * @return the logger
    */
   public static Log getLog(Class<?> theClass) {
-    logDirsCreateIfNotDone();
```
Comment (scalding) [inline grouper/src/grouper/edu/internet2/middleware/grouper/util/GrouperUtil.java:1125]: might need to uncomment

**Comment by cer28** [comment]
This code is now in a separate project: https://github.internet2.edu/internet2/grouper-ext-auth/tree/move-wip . I think this can be closed without merging.

**Comment by scalding** [comment]
Closing since this PR was against an old version of Grouper and the code has moved elsewhere, as noted.

**Pull request closed without merging**